doc/references.rst

```diff
@@ -0,0 +1,129 @@
+How to Cite
+===========
+
+If you like SpikeInterface, please star us on `Github <https://github.com/SpikeInterface/spikeinterface>`_!
+*giving us a star gives a measure of the level of use and interest, which goes a long way to getting funding*
+
+Please cite SpikeInterface in your papers with our eLife paper: [Buccino]_
+
+SpikeInterface stands on the shoulders of giants!
+Each method in SpikeInterface draws on (or directly runs) independently-created methods.
+Please try to reference the individual works that are important for your analysis pipeline.
+If you notice a missing reference, please let us know by submitting an issue on Github.
```

Comment:
```suggestion
If you notice a missing reference, please let us know by `submitting an issue <https://github.com/SpikeInterface/spikeinterface/issues/new` _ on Github.
```

Reply:
@h-mayorquin missing a space between the before the `_` right?

Reply:
should be good now!

Reply:
Thanks.

Reply:
This is still not displayed in the latest version : /

Reply:
It is on my side! Weird!